### PR TITLE
Providers connection improvements for Golden Age

### PR DESCRIPTION
**Pull request by ArtemBurakov**

### CODE REVIEW

File: provider-bridge-shared/constants.ts
Comment (rileystephens28) [inline provider-bridge-shared/constants.ts:1]: We should keep the value of `PELAGUS_WINDOW_PROVIDER_INJECTED_NAMESPACE` as "pelagus" as this version of pelagus will still be used after golden age and we want to dapps built during golden age to be compatible with mainnet.

File: src/window-provider.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {
 } from "@pelagus-provider/provider-bridge-shared"
 import PelagusWindowProvider from "@pelagus-provider/window-provider"
 
-const PELAGUS_OBJECT_PROPERTY = "pelagus"
+const PELAGUS_OBJECT_PROPERTY = "pelagusGoldenAge"
```
Comment (rileystephens28) [inline src/window-provider.ts]: We should keep the value of `PELAGUS_OBJECT_PROPERTY` as "pelagus" as well.

**Comment by ArtemBurakov** [comment]
Yes, I understand. I did this because the old wallet already has such a newspace and this will lead to conflicts. And if I change or remove the pelagus provider in the old version, it will lead to problems with existing apps and services that use it. So I had the only option to change it in the new version.

**Comment by ArtemBurakov** [comment]
If in any case we need to keep **pelagus** in the new version, you need to remove it in the old version. Providers of the old version will remain in _window.ethereum.providers_, but the provider for the old version in _window.pelagus_ will be replaced by the new provider. Accordingly, for the old version you will need to access _window.tally_ or map the _window.ethereum.providers_ array and look for **version 1**.

**Comment by ArtemBurakov** [comment]
I tried not to touch the old wallet because I was thinking about compatibility with dapps. So, if this older pelagus provider is not used in dapps - we can replace it with a new golden age pelagus provider.

**Comment by rileystephens28** [comment]
With the release of all of our new developer tools, we will push dapp builders to update to the new tool set so that their dapps are compatible when golden age testnet is launched. This will force builders to either update their apps or face deprecation issues, which I am okay with as it gives builders a compelling reason to update. 

Regarding the implementation in the claims dashboard, chances are high that by the time the dapp is available that the vast majority of iron age pelagus extension users will have the latest version due to chrome's auto extension updates. Meaning that we likely wont experience both versions of pelagus trying to write to `window.pelagus`. We can also implement some pretty simple logic to check that the user **is** using the latest version of iron age pelagus and if they arent force them to update before proceeding.

**Review by rileystephens28 — APPROVED** [review]

**Pull request merged**